src/main/scala/com/sksamuel/scapegoat/Inspection.scala

```diff
@@ -86,13 +93,11 @@ case class InspectionContext(global: Global, feedback: Feedback) {
       tree match {
         // ignore synthetic methods added
         case DefDef(mods, _, _, _, _, _) if tree.symbol.isSynthetic =>
-        case dd @ DefDef(_, _, _, _, _, _) if isSuppressed(dd.symbol) =>
```

Comment:
The three case clauses I deleted from this `match` expression are all descendants of `MemberDef`, so they are no longer needed once we handle that case.